TrackerConditions/inc/StrawElectronics.hh

```diff
@@ -180,8 +177,7 @@ namespace mu2e {
     void setAnalogNoise(std::array<double,npaths> analognoise) 
               { _analognoise = analognoise; }
     void setvthresh(std::vector<double> vthresh) { _vthresh = vthresh;}
-    void setFlashTDC( double flashStartTDC, double flashEndTDC) {
-      _flashStartTDC = flashStartTDC;
+    void setFlashTDC( double flashEndTDC) {
       _flashEndTDC = flashEndTDC;
```

Comment:
Who is responsible for setting this value?

Reply:
flashEnd is set in fcl in ns, flashEndTDC is just the conversion of that number into TDC counts as it would actually be set by slow controls